docs/reference/quickstart/full-text-filtering-tutorial.asciidoc

```diff
@@ -0,0 +1,588 @@
+----
+// TEST
+
+[discrete]
+[[full-text-filter-tutorial-match-query]]
+=== Step 3: Perform basic full-text searches
+
+Full-text search involves executing text-based queries across one or more document fields.
+These queries calculate a relevance score for each matching document, based on how closely the document's content aligns with the search terms.
+{es} offers various query types, each with its own method for matching text and <<relevance-scores,relevance scoring>>.
+
+[discrete]
+==== `match` query
+
+The <<query-dsl-match-query, `match`>> query is the standard query for full-text, or "lexical", search.
+The query text will be analyzed according to the analyzer configuration specified on each field (or at query time).
+
+First, search the `description` field for "fluffy pancakes":
+
+[source,console]
+----
+GET /cooking_blog/_search
+{
+  "query": {
+    "match": {
+      "description": {
+        "query": "fluffy pancakes", <1>
+        "analyzer": "standard" <2>
+      }
+    }
+  }
+}
+----
+// TEST[skip:Flakey test]
+<1> By default, the `match` query uses `OR` logic between the resulting tokens. This means it will match documents that contain either "fluffy" or "pancakes", or both, in the description field.
+<2> {es} defaults to the analyzer defined in the field mapping, so this field is unnecessary here. This is only required if you're using a different analyzer at search time, which is an <<different-analyzers,advanced use case>>.
+
+.Example response
+[%collapsible]
+==============
+[source,console-result]
+----
+{
+  "took": 2,
+  "timed_out": false,
+  "_shards": {
+    "total": 5,
+    "successful": 5,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": { <1>
+    "total": {
+      "value": 1,
+      "relation": "eq"
+    },
+    "max_score": 0.39556286, <2>
+    "hits": [
+      {
+        "_index": "cooking_blog",
+        "_id": "1",
+        "_score": 0.39556286, <3>
+        "_source": {
+          "title": "Perfect Pancakes: A Fluffy Breakfast Delight", <4>
+          "description": "Learn the secrets to making the fluffiest pancakes you've ever tasted. This recipe uses buttermilk and a special folding technique to create light, airy pancakes that are perfect for lazy Sunday mornings.", <5>
+          "author": "Maria Rodriguez",
+          "date": "2023-05-01",
+          "category": "Breakfast",
+          "tags": [
+            "pancakes",
+            "breakfast",
+            "easy recipes"
+          ],
+          "rating": 4.8
+        }
+      }
+    ]
+----
+// TEST[continued]
+<1> The `hits` object contains the total number of matching documents and their relation to the total. <1> The `relation` field can have one of three values: "eq", "gte", or "lte".
+* "eq": The `value` is exactly equal to the total number of hits.
+* "gte": The `value` is greater than or equal to the total number of hits. This occurs when the number of hits exceeds the configured max result window.
```

Comment:
This may be a little misleading. First of all, users might not know what the configured max result window is, second, this can be different depending on properties such as `track_total_hits`. 

I think the summary in the [track_total_hits](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-your-data.html#track-total-hits) doc is more correct: 

> When set to true the search response will always track the number of hits that match the query accurately (e.g. total.relation will always be equal to "eq" when track_total_hits is set to true). Otherwise the "total.relation" returned in the "total" object in the search response determines how the "total.value" should be interpreted. A value of "gte" means that the "total.value" is a lower bound of the total hits that match the query and a value of "eq" indicates that "total.value" is the accurate count.

I wonder if this is a case where minimal explanation is better?

Reply:
> I wonder if this is a case where minimal explanation is better?

Definitely wandered deep into overkill territory here I think

Reply:
I vote for removing the list and explanations of `relation` and linking to track total hits for those interested 👍